clipboard/clipboard.d.ts

```diff
@@ -3,8 +3,8 @@
 // Definitions by: Andrei Kurosh <https://github.com/impworks>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare class Clipboard {
-    constructor(selector: (string | Element | NodeListOf<Element>), options?: ClipboardOptions);
+interface ClipboardStatic {
+    new(selector: (string | Element | NodeListOf<Element>), options?: ClipboardOptions): ClipboardStatic;
```

Comment:
From documentation: https://github.com/zenorocha/clipboard.js#events

```ts
var clipboard = new Clipboard('.btn');

clipboard.on('success', function(e) {
    console.info('Action:', e.action);
    console.info('Text:', e.text);
    console.info('Trigger:', e.trigger);

    e.clearSelection();
});

clipboard.on('error', function(e) {
    console.error('Action:', e.action);
    console.error('Trigger:', e.trigger);
});
```

that tells me that the current declaration file is accurate.